test/uinta/formatter_test.exs

```diff
@@ -4,23 +4,35 @@ defmodule Uinta.FormatterTest do
 
   test "formats message properly" do
     message = "this is a log message"
-    timestamp = {{2020, 3, 16}, {10, 16, 32, 548}}
+    date = {2020, 3, 16}
+    time = {10, 16, 32}
+    timestamp = {date, Tuple.append(time, 548)}
     metadata = [request_id: "req_1234", user_uid: "26dbba1d-5b72-4e5c-b1a7-701589343291"]
 
     formatted = Formatter.format(:info, message, timestamp, metadata)
 
+    {_date, {hour, _m, _s}} = :calendar.local_time_to_universal_time({date, time})
```

Comment:
Can you help me understand what this change does? Does it make the test work for different time zones?

Reply:
it breaks due to DST :/  matter of fact -> local_time_to_universal_time is deprecated but the local_time_to_universal_time_dst one adds more complication since it s possible ot have an invalid time which would mess with our logs